source/slang/hlsl.meta.slang

```diff
@@ -717,12 +717,20 @@ __generic<T : __BuiltinFloatingPointType, let N : int> vector<T,N> exp2(vector<T
 __generic<T : __BuiltinFloatingPointType, let N : int, let M : int> matrix<T,N,M> exp2(matrix<T,N,M> x);
 
 // Convert 16-bit float stored in low bits of integer
+__target_intrinsic(glsl, "unpackHalf2x16($0).x")
 float f16tof32(uint value);
-__generic<let N : int> vector<float,N> f16tof32(vector<uint,N> value);
+
+__generic<let N : int>
+__target_intrinsic(glsl, "unpackHalf2x16($0).x")
+vector<float,N> f16tof32(vector<uint,N> value);
```

Comment:
As discussed separately, it would be good to align the naming/signature of these operations to match with what dxc has added for 16-bit type support. I'm fine with these for now, and they might be useful to keep around just because they have such a direct mapping to GLSL.